.gitignore

```diff
@@ -22,3 +22,4 @@ cabal.project.local~
 .HTF/
 .ghc.environment.*
 result*
+.vscode/
```

Comment:
minor improvement (outside the scope of the issue been addressed)

Reply:
Ooh, could we also add `.venv`? I'm working on some Python stuff for Inferno but it's going to take a while before I can open a PR, and my vscode goes nuts every time I switch back to the master branch 😅

Reply:
sure!